tests/loadtests/glloadtests/resources/mac/Info.plist

```diff
@@ -5,17 +5,17 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>${PRODUCT_NAME}</string>
+	<string>gl3loadtests</string>
 	<key>CFBundleExecutable</key>
-	<string>${EXECUTABLE_NAME}</string>
+	<string>gl3loadtests</string>
 	<key>CFBundleIconFile</key>
 	<string>ktx_app.icns</string>
 	<key>CFBundleIdentifier</key>
-	<string>${PRODUCT_BUNDLE_IDENTIFIER}</string>
+	<string>org.khronos.ktx.gl3loadtests</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>${PRODUCT_NAME}</string>
+	<string>gl3loadtests</string>
```

Comment:
I first tried to do so, but struggled to get CMake to replace it properly. Since the project already had dedicated files for glloadtests and vkloadtests respectively, I changed this to a static value to have a quick solution.
We can try to improve this once all important things are running again.